src/gui/guiTable.cpp

```diff
@@ -60,6 +60,8 @@ GUITable::GUITable(gui::IGUIEnvironment *env,
 		m_rowheight = MYMAX(m_rowheight, 1);
 	}
 
+	m_vertical_image_padding = 4.0f * RenderingEngine::getDisplayDensity();
```

Comment:
This should probably become an option in `tablecolumns[]` ... but then I'd have to test it somehow

Reply:
Added the `vpadding` option and extended the Devtest test formspec